src/__tests__/starWarsSchema.js

```diff
@@ -170,6 +170,29 @@ const humanType = new GraphQLObjectType({
         'The friends of the human, or an empty list if they have none.',
       resolve: (human) => getFriends(human),
     },
+    friendsAsync: {
+      type: GraphQLList(characterInterface),
+      description:
+        'The friends of the droid, or an empty list if they have none. Returns an AsyncIterable',
+      args: {
+        errorIndex: { type: GraphQLInt },
```

Comment:
It's better to move non-trivial tests here: https://github.com/graphql/graphql-js/blob/master/src/execution/__tests__/lists-test.js

Starwars schema is intended as just a sample schema to quickly demonstrate all basic features.

Reply:
Removed these tests and added new tests to `lists-test.js`